src/Cli/dotnet/commands/dotnet-test/MSBuildHandler.cs

```diff
@@ -99,15 +113,6 @@ private void InitializeTestApplications(IEnumerable<TestModule> modules)
                     throw new InvalidOperationException();
```

Comment:
Comment above this line should say `TestModule` per the rename. Sorry it seems I forgot to update the comment when I did the rename.